sos/policies/__init__.py

```diff
@@ -429,44 +432,41 @@ def display_results(self, archive, directory, checksum, archivestat=None,
                          file for this run
         :type map_file: ``str``
         """
-        # Logging is already shutdown and all terminal output must use the
-        # print() call.
+        # Logging is shut down, but there are some edge cases where automation
+        # does not capture printed output (e.g. avocado CI). Use the ui_log to
+        # still print to console in this case.
 
         # make sure a report exists
         if not archive and not directory:
             return False
 
-        self._print()
-
         if map_file:
-            self._print(_("A mapping of obfuscated elements is available at"
-                          "\n\t%s\n" % map_file))
+            self.ui_log.info(
+                _(f"\nA mapping of obfuscated elements is available at"
+                  f"\n\t{map_file}")
+            )
 
         if archive:
-            self._print(_("Your sosreport has been generated and saved "
-                          "in:\n\t%s\n") % archive, always=True)
-            self._print(_(" Size\t%s") %
-                        get_human_readable(archivestat.st_size))
-            self._print(_(" Owner\t%s") %
-                        getpwuid(archivestat.st_uid).pw_name)
+            self.ui_log.info(
+                _(f"\nYour sosreport has been generated and saved in:"
+                  f"\n\t{archive}\n")
+            )
+            self.ui_log.info(
+                _(f" Size\t{get_human_readable(archivestat.st_size)}")
+            )
+            self.ui_log.info(
+                _(f" Owner\t{getpwuid(archivestat.st_uid).pw_name}")
+            )
         else:
-            self._print(_("Your sosreport build tree has been generated "
-                          "in:\n\t%s\n") % directory, always=True)
+            self.ui_log.info(
+                _(f"Your sosreport build tree has been generated in:"
```

Comment:
There should be leading `\n`, since we deleted blank `self._print()`. In `if archive` branch, we added it properly.

(.. or have the blank `self.ui_log.info("")` as a full replacement of the `self._print()`)

Reply:
That was intentional - new output is:

```
  Finished running plugins

Your sosreport build tree has been generated in:
	/var/tmp/sosreport-terra-2023-01-23-uceowet

```

Rather than 2 blank lines between the `Finished` line and this.